### DISCUSSION TITLE
Custom sub folders option

### DISCUSSION THREAD

**Discussion by coldasicee** [BlafKing/sd-civitai-browser-plus#282] [Ideas]
I wanted to add my LoRas to folders like characters/styles/nsfw, etc. Is there a way to add the option to create custom sub folders?

**Comment by BlafKing** [comment]
There is!

You can add custom sub folders by creating the folder(s) in the content type that you'd like to have them in.
You can either do this by doing it in the file manager, or by editing the text in "Download folder".

For example, changing the download folder text from `webui\models\Stable-diffusion` to `webui\models\Stable-diffusion\custom\path`

After creating the folder or after downloading a model to the folder you'd like then it should show up in the sub folder options! :)
